### ISSUE TITLE
not punishing negative samples？

### ISSUE THREAD

**Issue by bruceyang2012** [xingyizhou/CenterNet#60]
If there is no target object in the training picture, will this picture be considered in the training process? I added some negative samples without targets in the training set, and the model performed well on the positive samples, but also produced a lot of wrong predictions on the negative samples. Therefore, I suspect that the model does not punish negative samples in the training process. How can I solve this problem?

**Comment by xingyizhou** [comment]
Yes, the image will be supervised to produce 0 heatmap. You can add `--debug 2` to visualize the ground truth / predicted heatmap. You will need to add the basic dataset information [here](https://github.com/xingyizhou/CenterNet/blob/master/src/lib/utils/debugger.py#L25) if you are working on your own dataset.

**Comment by bruceyang2012** [comment]
@xingyizhou In my experiment, when the negative sample is added to the training set, a large number of false positives will still be generated on the negative sample. I increase the loss weight of the negative sample, and it still doesn't improve. What is the reason and how can I reduce the false positive on the negative sample?

**Comment by bhack** [comment]
Is it needed some form of hard negative sample mining?

https://arxiv.org/abs/1808.04285

**Comment by Wenboisstrong** [comment]
@bruceyang2012  I'm also interested in this question. Do you have any progress now?

**Comment by qiansgithub** [comment]
me too,do you have any solutions now?

**Comment by developer0hye** [comment]
@bruceyang2012 

Hi, brcueyang!
Do you solve this problem?
